src/connectors/rating-menu/__tests__/connectRatingMenu-test.js

```diff
@@ -278,13 +271,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/rating-menu
 
     // Second rendering
     expect(helper.getRefinements(attribute)).toEqual([
-      { type: 'disjunctive', value: '3' },
-      { type: 'disjunctive', value: '4' },
-      { type: 'disjunctive', value: '5' },
+      { operator: '>=', type: 'numeric', value: [3] },
     ]);
     refine('3');
-    expect(helper.getRefinements(attribute)).toEqual([]);
-    expect(helper.state.disjunctiveFacetsRefinements).toEqual({ swag: [] });
+    expect(helper.getRefinements(attribute)).toEqual([
+      { operator: '>=', type: 'numeric', value: [] },
```

Comment:
why not an empty array? is it the way helper works?

Reply:
Yes, this is how the helper is resetting numeric refinements.